source/install.txt

```diff
@@ -154,6 +154,23 @@ Install Compass
             to modify your system settings to grant |compass-short|
             permissions to run. You may be prompted to enter your
             system password before launching |compass-short|.
+         
+         
+         7. Allow macOS to trust |compass-short|. If you receive a
```

Comment:
Also, this is slightly out of scope for this specific ticket, but the current screenshot of the "Downloaded from the Internet" dialog is dated 2015, which may give off the impression that these docs haven't been updated for 5 years (not true, but it may come off that way).

I'd suggest as part of this change that we replace that step with something that specifically describes the dialog box and remove the screenshot. For example:

```
#. When you open |compass| for the first time, you may receive a notice stating
   that it is an application downloaded from the internet, requiring you to confirm
   you want to open it. Click :guilabel:`Open` to continue and launch |compass-short|.
```